Assets/UI/actionpanel.lua

```diff
@@ -1079,44 +1080,52 @@ end
-    -- this boolean and will prevent spam clicking through; as the tutorial
-    -- system itself sets ENABLED as it's hiding all controls.
-    if m_isSlowTurnEnable then
-        Controls.TutorialSlowTurnEnableAnim:SetHide(false);
-        Controls.TutorialSlowTurnEnableAnim:SetToBeginning();
-        Controls.TutorialSlowTurnEnableAnim:RegisterEndCallback(
-            function()
-                Controls.TutorialSlowTurnEnableAnim:SetHide(true);
-            end
-        );
-        Controls.TutorialSlowTurnEnableAnim:Play();
-    end
-    ContextPtr:RequestRefresh();
+    -- If the player is dead (observing), then automatically end their turn
+    local pPlayerConfig :table	= PlayerConfigurations[Game.GetLocalPlayer()];
+    if (not pPlayerConfig:IsAlive()) then
+        DoEndTurn();
+    else
+        -- Standard disable is set to false in the refresh.
+        -- This extra level of input catching is done when tutorial has raised
+        -- this boolean and will prevent spam clicking through; as the tutorial
+        -- system itself sets ENABLED as it's hiding all controls.
+        if m_isSlowTurnEnable then
+            Controls.TutorialSlowTurnEnableAnim:SetHide(false);
+            Controls.TutorialSlowTurnEnableAnim:SetToBeginning();
+            Controls.TutorialSlowTurnEnableAnim:RegisterEndCallback(
+                function()
+                    Controls.TutorialSlowTurnEnableAnim:SetHide(true);
+                end
+            );
+            Controls.TutorialSlowTurnEnableAnim:Play();
+        end
+        ContextPtr:RequestRefresh();
 
-    -- if auto-cycle is OFF, play this sound to indicate "start of turn"
-    if (not UserConfiguration.IsAutoUnitCycle()) then
-            UI.PlaySound("SP_Turn_Start");
--- ==== CQUI CUSTOMIZATION BEGIN  ==================================================================================== --
-            -- AZURENCY : also reset the policy reminder shown status
-            CQUI_PolicyReminderClosed = false
--- ==== CQUI CUSTOMIZATION END  ====================================================================================== -- 
+        -- if auto-cycle is OFF, play this sound to indicate "start of turn"
+        if (not UserConfiguration.IsAutoUnitCycle()) then
+                UI.PlaySound("SP_Turn_Start");
+            -- ==== CQUI CUSTOMIZATION BEGIN  ==================================================================================== --
+                -- AZURENCY : also reset the policy reminder shown status
+                CQUI_PolicyReminderClosed = false
+            -- ==== CQUI CUSTOMIZATION END  ====================================================================================== -- 
+        end
     end
 end
 
 -- ===========================================================================
 function OnLocalPlayerTurnEnd()
+    local pPlayerConfig :table	= PlayerConfigurations[Game.GetLocalPlayer()];
+    if (pPlayerConfig:IsAlive()) then
```

Comment:
again the Git compare is not doing a good job
everything from line 1111 to line 1119 on the LEFT side matches everything from line 1119 to line 1127 on the RIGHT side